Integrations/Gmail/Gmail.py

```diff
@@ -167,6 +169,16 @@ def parse_mail_parts(parts):
     return body, html, attachments
 
 
+def parse_privileges(privileges_list):
+    privileges = []
+    for prvlg in privileges_list:
+        service_id = prvlg.get('serviceId')
+        name = prvlg.get('privilegeName')
+        if name and service_id:
```

Comment:
please explain this condition

Reply:
Well, there's no good explanation rather than checking the validity of the returned data. We can delete this condition, what do you say? That's what returned.
![image](https://user-images.githubusercontent.com/53565845/70851848-3d968a80-1ea3-11ea-9a57-dd0c43bf7d46.png)

Reply:
what i dont understand is why we need to check both exist ? why one of the id or name is enough ?

Reply:
I'll change it